README.md

```diff
@@ -2,6 +2,13 @@
 
 Built a calculator for Math-magicians Inc using create-react-app. Math-magicians is a fictional company based in France. They have requested help to build a simple product that would assist them with their calculations.
 
+## Heroku link
+
+[leon's calc](https://leon-calc.herokuapp.com)
```

Comment:
- [ ] [REQUIRED] Please, make sure that your live demo is up to date. Keep in mind that your readme is the first thing users check.